src/lib.rs

```diff
@@ -707,10 +705,10 @@ impl Player {
             }
         }
 
-        let is_subtitle_cyclable = self.subtitle_stream_info.1 > 1;
-        let is_audio_cyclable = self.audio_stream_info.1 > 1;
+        let is_subtitle_cycleable = self.subtitle_stream_info.1 > 1;
```

Comment:
i believe "cyclable" is the correct spelling, atleast [per wiktionary](https://en.wiktionary.org/wiki/cyclable)